packages/interaction-viewer/package.json

```diff
@@ -11,6 +11,10 @@
     "uniprot",
     "ebi"
   ],
+  "files": [
+    "src",
+    "dist"
```

Comment:
I just read the files part in https://docs.npmjs.com/files/package.json 

> Files included with the “package.json#files” field cannot be excluded through .npmignore or .gitignore.

but we have dist in the git ignore, is that going to be an issue?

Reply:
We've had this in most modules for a while now, I guess what happens is that at the time of publishing they get included regardless. According to this we should remove `/dist` from `.gitignore`?

Reply:
I think the current effect that we have is that `/dist` gets included in npm but is not included in github, which I think is what we want.
So I guess, just leave it there.